src/persistence/local-storage/index.ts

```diff
@@ -298,14 +315,14 @@ export function removePendingDepositByHash(
       [hermezEthereumAddress]: deposits.filter((deposit) => deposit.hash !== hash),
     },
   };
-  setStorageKey(constants.PENDING_DEPOSITS_KEY, newStorage);
+  setStorageKey(constants.PENDING_DEPOSITS_KEY, JSON.stringify(newStorage));
```

Comment:
Might bee worth reconsidering this. It kinda sucks we can't abstract it just for 1 storage element that isn't an object

Reply:
I think it's ok as it's now, `setStorageKey` accepts a `string` so if you already have a string you can send it straight, if you have an object you use `JSON.stringify` and if you have a number you can use `toString`.

> It kinda sucks we can't abstract it just for 1 storage element that isn't an object

We don't need to abstract it, having just 1 storage element that isn't an object may change tomorrow. This feels like the correct implementation to me.

Reply:
As you see better :)